pkg/services/sqlstore/migrations/accesscontrol/dashboard_permissions.go

```diff
@@ -294,3 +295,120 @@ func (m *managedFolderCreateAction) Exec(sess *xorm.Session, migrator *migrator.
 	}
 	return nil
 }
+
+const managedFolderAlertActionsMigratorID = "managed folder permissions alert actions migration"
+
+func AddManagedFolderAlertActionsMigration(mg *migrator.Migrator) {
+	mg.AddMigration(managedFolderAlertActionsMigratorID, &managedFolderAlertActionsMigrator{})
+}
+
+type managedFolderAlertActionsMigrator struct {
+	migrator.MigrationBase
+}
+
+func (m *managedFolderAlertActionsMigrator) SQL(dialect migrator.Dialect) string {
+	return CodeMigrationSQL
+}
+
+func (m *managedFolderAlertActionsMigrator) Exec(sess *xorm.Session, mg *migrator.Migrator) error {
+	var ids []interface{}
+	if err := sess.SQL("SELECT id FROM role WHERE name LIKE 'managed:%'").Find(&ids); err != nil {
+		return err
+	}
+
+	if len(ids) == 0 {
+		return nil
+	}
+
+	var permissions []accesscontrol.Permission
+	if err := sess.SQL("SELECT role_id, action, scope FROM permission WHERE role_id IN(?"+strings.Repeat(" ,?", len(ids)-1)+") AND scope LIKE 'folders:%'", ids...).Find(&permissions); err != nil {
```

Comment:
I would recommend using DSL that xorm provides. It will be less prone to errors. The query would look something like
`sess.Table(accesscontrol.Permission{}).Where("scope LIKE folders:%").In("role_id", ids)`